### PR TITLE
Add `grandamento:execute`, `grandamento:get-buy-amount`, small fixes, and some Granda Mento wrapper fns

### PR DESCRIPTION
**Pull request by tkporter**
### Description

* Adds `grandamento:execute` to the cli, which executes an approved exchange proposal that has passed the veto period since the time of approval
* Adds `grandamento:get-buy-amount` to the cli, which calculates the amount of the buy token a proposal would receive according to the current oracle price & the spread
* Adds some wrapper functions to the contractkit GrandaMento wrapper
* Removes some unnecessary `await`s that my editor found
* Ensures all the `grandamento` CLI commands run their checks

### Other changes

n/a

### Tested

Unit tests, also ran the commands

### Related issues

n/a

### Backwards compatibility

Fully backward compatible

### Documentation

Autogenerated CLI docs

### CODE REVIEW

File: packages/cli/src/commands/grandamento/cancel.test.ts [hunk 1]
```diff
@@ -16,13 +16,9 @@ testWithGanache('grandamento:cancel cmd', (web3: Web3) => {
...
 
   const increaseLimits = async () => {
-    await (
-      await grandaMento.setStableTokenExchangeLimits(
-        'StableToken',
-        newLimitMin.toString(),
-        newLimitMax.toString()
-      )
-    ).sendAndWaitForReceipt()
+    await grandaMento
```
Comment (martinvol) [inline packages/cli/src/commands/grandamento/cancel.test.ts:19]: Odd the lint didn't catch this

File: packages/cli/src/commands/grandamento/cancel.ts [hunk 2]
```diff
@@ -26,11 +26,11 @@ export default class Cancel extends BaseCommand {
     const res = this.parse(Cancel)
     const proposalID = res.flags.proposalID
 
-    await newCheckBuilder(this).grandaMentoProposalExists(proposalID)
+    await newCheckBuilder(this).grandaMentoProposalExists(proposalID).runChecks()
```
Comment (martinvol) [inline packages/cli/src/commands/grandamento/cancel.ts:29]: Nice catch

File: packages/sdk/contractkit/src/wrappers/GrandaMento.ts [hunk 3]
```diff
@@ -58,17 +59,36 @@ export interface ExchangeProposalReadable {
...
+
+  maxApprovalExchangeRateChange = proxyCall(
+    this.contract.methods.maxApprovalExchangeRateChange,
+    undefined,
+    fixidityValueToBigNumber
+  )
+  setMaxApprovalExchangeRateChange = proxySend(
+    this.kit,
+    this.contract.methods.setMaxApprovalExchangeRateChange
+  )
```
Comment (martinvol) [inline packages/sdk/contractkit/src/wrappers/GrandaMento.ts:76]: Only nit would we adding a simple tests for this getters

File: packages/sdk/contractkit/src/wrappers/GrandaMento.ts [hunk 4]
```diff
@@ -173,6 +194,17 @@ export class GrandaMentoWrapper extends BaseWrapper<GrandaMento> {
     return out
   }
 
+  async getBuyAmount(
```
Comment (martinvol) [inline packages/sdk/contractkit/src/wrappers/GrandaMento.ts:198]: ditto

**Review by martinvol — APPROVED** [review]

File: packages/cli/src/commands/grandamento/get-buy-amount.ts [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+import { Flags } from '../../utils/command'
+import { enumEntriesDupWithLowercase } from '../../utils/helpers'
+
+const stableTokenOptions = enumEntriesDupWithLowercase(Object.entries(StableToken))
+
+export default class GetBuyAmount extends BaseCommand {
+  static description = 'Gets the buy amount for a prospective Granda Mento exchange'
+
+  static flags = {
+    ...BaseCommand.flags,
```
Comment (martinvol) [inline packages/cli/src/commands/grandamento/get-buy-amount.ts:15]: Tests are failing with `Missing required flag:  --from FROM  Account Address`
```suggestion
```

Comment (tkporter) [reply packages/cli/src/commands/grandamento/get-buy-amount.ts:15]: Battled tests for like 2 days straight- that was a bit of a red herring, that particular log was from a different test. Eventually got all the tests working though!
`BaseCommand.flags` doesn't include `--from` and is useful particularly for `--node`

**Pull request merged**